app/views/events/show.html.erb

```diff
@@ -0,0 +1,44 @@
+
+<%= render "shared/header"%>
+
+<h1>
+  <strong>Title:</strong>
+  <%= @event.title%>
+</h1>
+
+<p>
+  <strong>Description:</strong>
+  <%= @event.description %>
+</p>
+
+<p>
+  <strong>Creator:</strong>
+  <%= @event.creator.name %>
+</p>
+
+<p>
+  <strong>Date:</strong>
+  <%= @event.date %>
+</p>
+
+
+<h1>Attending:</h1>
+<% @event.attendees.each do |attendee| %>
+
+  <div class="content">
+    <small><%= attendee.name %></small><br />
+    <small><%= attendee.email %></small><br />
+  </div>
+
+<% end %>
+
+<% if logged_in? %>
+  <% if @event.attendees.include? current_user %>
+    <%= button_to 'Withdraw', withdraw_event_path(@event), method: :post %>
+  <% else %>
+    <%= button_to 'Attend', attend_event_path(@event), method: :post %>
+  <% end %>
+<% end %>
```

Comment:
* [ ] There should be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* In [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) explains how to put if/else statements within helpers so let's do the same for your code. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright